src/v/serde/serde.h

```diff
@@ -46,8 +46,6 @@
 
 namespace serde {
 
-using checksum_t = uint32_t;
```

Comment:
To make b2ae8d6d930a84d3ddfa970fb0c05a2348928581 compile we'll need to include `checksum_t.h` somewhere right?

Reply:
Added `#include "serde/checksum_t.h`.